en-US/Players_Guide/03_Creating_a_Character/Equipment/Mounts_and_Vehicles.md

```diff
@@ -22,12 +22,6 @@ An exotic saddle is required for riding any aquatic or flying mount.
 **Vehicle Proficiency.**
 If you have proficiency with a certain kind of vehicle (land or water), you can add your proficiency bonus to any check you make to control that kind of vehicle in difficult circumstances.
 \
```

Comment:
The slash should be removed with an empty line; it adds a manual line break, that we no longer need here.